common/trainers/classification_trainer.py

```diff
@@ -10,10 +10,10 @@
 from .trainer import Trainer
 
 
-class ReutersTrainer(Trainer):
+class ClassificationTrainer(Trainer):
 
     def __init__(self, model, embedding, train_loader, trainer_config, train_evaluator, test_evaluator, dev_evaluator):
-        super(ReutersTrainer, self).__init__(model, embedding, train_loader, trainer_config, train_evaluator, test_evaluator, dev_evaluator)
+        super(ClassificationTrainer, self).__init__(model, embedding, train_loader, trainer_config, train_evaluator, test_evaluator, dev_evaluator)
```

Comment:
`super().__init__(...)` is sufficient in Python 3.

Reply:
Agreed. Fixed in ecda4c0409cc5f219eac0873e42b79de79f79861